docs/expressions/syntax.md

```diff
@@ -24,6 +24,7 @@ the lower level syntax of math.js. Differences are:
   not bitwise xor.
 - Implicit multiplication, like `2 pi`, is supported and has special rules.
 - Relational operators (`<`, `>`, `<=`, `>=`, `==`, and `!=`) are chained, so the expression `5 < x < 10` is equivalent to `5 < x and x < 10`.
+- The precedence of the nullish coalescing operator `??` is higher than in JavaScript. For example, `2 ^ x ?? 0` is grouped as `2 ^ (x ?? 0)`.
```

Comment:
As I suspected, other precedences are different as well. For example, in the mathjs expression language exponentiation is higher precedence than prefix operators like `-`, whereas in JavaScript, their relative precedence is undefined because any expression in which the precedence would matter is a syntax error. So please either change this to "The precedence of some operators, such as ?? (nullish coalescing), is different than in JavaScript" or "The precedence of some operators is different; see a complete list at the precedence table below." and add by the table a list of all the differences based on a careful reading of the MathJS and JavaScropt precedence tables. Thanks!